src/libknot/yparser/ypbody.c

```diff
@@ -104,7 +104,7 @@ static const unsigned char _yparser_index_offsets[] = {
 	195
 };
 
-static const char _yparser_indicies[] = {
+static const char _yparser_indices[] = {
```

Comment:
If certain files shouldn't be changed, let me know

Reply:
This is a file generated by an external tool. Keep the original one.